v2/concordium/types.proto

```diff
@@ -692,9 +702,227 @@ message ConsensusInfo {
 }
 
 // Information about an arrived block that is part of the streaming response.
+message ArrivedBlockInfo {
+  // Hash of the block.
+  BlockHash hash = 1;
+  // Absolute height of the block, height 0 is the genesis block.
+  AbsoluteBlockHeight height = 2;
+}
+
+// The response for GetCryptographicParameters.
+message CryptographicParameters {
+  // A free-form string used to distinguish between different chains even if they share other parameters.
+  string genesis_string = 1;
+  // Generators for the bulletproofs.
+  bytes bulletproof_generators = 2;
```

Comment:
We could specify that this is a list of 256 group elements in the G1 group of the BLS12-381 curve.